src/tests/suite/daos_container.c

```diff
@@ -653,6 +655,25 @@ co_acl(void **state)
 
 	co_acl_get(arg, exp_acl, exp_owner, exp_owner_grp);
 
+	print_message("Case 4: delete entry from ACL\n");
+
+	/* Update expected ACL to remove the entry */
+	assert_int_equal(daos_acl_remove_ace(&exp_acl, type_to_remove,
+					     name_to_remove), 0);
+
+	rc = daos_cont_delete_acl(arg->coh, type_to_remove, name_to_remove,
+				  NULL);
+	assert_int_equal(rc, 0);
+
+	co_acl_get(arg, exp_acl, exp_owner, exp_owner_grp);
+
+	print_message("Case 5: delete entry no longer in ACL\n");
+
+	/* try deleting same entry again - should be gone */
+	rc = daos_cont_delete_acl(arg->coh, type_to_remove, name_to_remove,
+				  NULL);
+	assert_int_equal(rc, -DER_NONEXIST);
```

Comment:
one possible addition (minor - only if you need to update something else in the PR), supply DAOS_HDL_INVAL as the first argument to daos_cont_delete_acl() and expect -DER_NO_HDL error returned.

Reply:
Good idea - I need to fix a conflict before I can merge, so I'll add this.